GTBitsOfGood/canine-assistants
# Forgot Password

## Forgot Password

Issue Number(s): #193 

The Forgot Password modal and backend functionality has been created to allow for the user to reset their password if forgotten after inputting a correct reset code sent to their email.

### Checklist


### Critical Changes

- New model created for `resetCode`, make sure to reset MongoDB if running into issues!

### Related PRs

PR Number(s): #193 

### How To Test

First log in as an Admin or Manager and invited yourself as a user using your real email address. Then Sign Up using that real email. Then log out again and click `Forgot Password?` on the Login screen. Make sure to test on mobile as well!
